server/schedule/operator_controller.go

```diff
@@ -769,7 +769,7 @@ func (oc *OperatorController) updateCounts(operators map[uint64]*operator.Operat
 		delete(oc.counts, k)
 	}
 	for _, op := range operators {
-		oc.counts[op.Kind()]++
+		oc.counts[op.SchedulerKind()]++
```

Comment:
Maybe we can implement a new `OperatorCount` function without changing the existing function?